docs/src/terminology.md

```diff
@@ -217,9 +217,17 @@ A 64-byte ed25519 signature of R (32-bytes) and S (32-bytes). With the requireme
 This requirement ensures no signature malleability. Each transaction must have at least one signature for [fee account](terminology#fee-account).
 Thus, the first signature in transaction can be treated as [transacton id](terminology.md#transaction-id)
 
+## skipped slot
+
+A past [slot](terminology.md#slot) that did not produce a [block](terminology.md#block), because the leader was offline or the [fork](terminology.md#fork) containing the slot was abandoned for a better alternative by cluster consensus. A skipped slot will not appear as an ancestor for blocks at subsequent slots, increment the [block height](terminology#block-height), nor expire the oldest `recent_blockhash`.
```

Comment:
```suggestion
A past [slot](terminology.md#slot) that did not produce a [block](terminology.md#block), because the leader was offline or the [fork](terminology.md#fork) containing the slot was abandoned for a better alternative by cluster consensus. A skipped slot will not appear as an ancestor for blocks at subsequent slots, nor increment the [block height](terminology#block-height), nor expire the oldest `recent_blockhash`.
```

Reply:
Sorry, missed word the first time.